src/System/Hapistrano/Config.hs

```diff
@@ -99,6 +104,32 @@ data Target =
     , targetSshArgs :: [String]
     } deriving (Eq, Ord, Show)
 
+data BuildCommand = BuildCommand
+  { buildCommandCommand  :: GenericCommand
+  , buildCommandOnlyLead :: ExecutionMode
```

Comment:
Hey @sestrella I think the name `buildCommandOnlyLead` can be misleading. It makes me think the value is a boolean rather than a `ExecutionMode`. What about renaming it to `buildCommandExecMode`?